src/main/java/com/pet/bankservice/security/jwt/JwtUser.java

```diff
@@ -0,0 +1,52 @@
+package com.pet.bankservice.security.jwt;
+
+import java.util.Collection;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+@AllArgsConstructor
+@Data
+public class JwtUser implements UserDetails {
```

Comment:
```suggestion
public class User implements UserDetails {
```
I guess there is no need to call the user as a JwtUser

Reply:
I don't like idea to have 2 classes with the same name, and also I don't want to change my User model so I would like to leave this as is